### PR TITLE
add 3 FPS games to the list

### PR DESCRIPTION
**Pull request by Tx-Charlie**
add:
* Red Eclipse
* Urban Terror
* Darkest Hour

### CODE REVIEW

**Comment by michelpereira** [comment]
Hi, @Tx-Charlie .
Based on the guidelines here it goes my recommendations for the descriptions:
**Redeclipse:** Fun-filled new take on the first person arena shooter, featuring parkour, impulse boosts, and more.
**Urban terror:** Free multiplayer first person shooter that will run on any Quake III Arena compatible engine.
**Darkest Hour:** World War 2 online multiplayer tactical shooter based on Red Orchestra: Ostfront.

Thanks.

File: README.md [hunk 1]
```diff
@@ -283,6 +283,7 @@ Set](https://github.com/juliangarnier/3D-Hartwig-chess-set) - 3D chess game done
 - [Ioquake3](https://github.com/ioquake/ioq3) - The free software FPS engine that can be used to play Quake 3, or make your own game.
 - [OpenLara](https://github.com/XProger/OpenLara) - Tomb Raider open-source engine.
 - [Openspades](https://github.com/yvt/openspades) - Compatible client of Ace of Spades 0.75.
+- [Red Eclipse](https://github.com/redeclipse) - A fun-filled new take on the first person arena shooter, featuring parkour, impulse boosts, and more.
```
Comment (michelpereira) [inline README.md:287]: `- [Red Eclipse](https://github.com/redeclipse) - Fun-filled new take on the first person arena shooter, featuring parkour, impulse boosts, and more.`

**Review by michelpereira — CHANGES_REQUESTED** [review]

**Comment by michelpereira** [comment]
@Tx-Charlie , I did some comments and please let me know when you submit again.

**Comment by Tx-Charlie** [comment]
> @Tx-Charlie , I did some comments and please let me know when you submit again.

Ok, done, removed the `A` in `red eclipse` 's description.

**Comment by michelpereira** [comment]
> > @Tx-Charlie , I did some comments and please let me know when you submit again.
> 
> Ok, done, removed the `A` in `red eclipse` 's description.

Hi, there. Did you see the other one comments that I made?

**Review by michelpereira — APPROVED** [review]

**Review by michelpereira — CHANGES_REQUESTED** [review]
- [Urban Terror](https://github.com/FrozenSand/UrbanTerror4) - Free multiplayer first person shooter that will run on any Quake III Arena compatible engine.

**Review by michelpereira — CHANGES_REQUESTED** [review]
`- [Darkest Hour](https://github.com/DarklightGames/DarkestHour) - World War 2 online multiplayer tactical shooter based on Red Orchestra: Ostfront.`

File: README.md [hunk 2]
```diff
@@ -278,6 +278,7 @@ Set](https://github.com/juliangarnier/3D-Hartwig-chess-set) - 3D chess game done
 ### FPS
 
 - [AssaultCube](https://github.com/assaultcube/AC) - Total conversion of Wouter van Oortmerssen's FPS called Cube.
+- [Darkest Hour](https://github.com/DarklightGames/DarkestHour) A World War 2 online multiplayer tactical shooter based on Tripwire Interactive’s award winning game Red Orchestra: Ostfront. (must own the copy of `Red Orchestra: Ostfront` for running this mod)
```
[the remark this answers is not in the document]
Comment (Tx-Charlie) [reply README.md]: > > > @Tx-Charlie , I did some comments and please let me know when you submit again.
> > 
> > 
> > Ok, done, removed the `A` in `red eclipse` 's description.
> 
> Hi, there. Did you see the other one comments that I made?

oops, sry, all fixed now.

**Review by michelpereira — APPROVED** [review]

**Review by michelpereira — APPROVED** [review]

**Review by michelpereira — CHANGES_REQUESTED** [review]
Sorry but there's a dash missing, please update to:
`- [Darkest Hour](https://github.com/DarklightGames/DarkestHour) - World War 2 online multiplayer tactical shooter based on Red Orchestra: Ostfront.`

**Comment by Tx-Charlie** [comment]
oops, sry

**Pull request merged**